### PR TITLE
Test: Replace NewClient() with NewClientT()

### PR DESCRIPTION
**Pull request by thaJeztah**
ping @vdemeester PTAL 🤗

### CODE REVIEW

File: integration-cli/docker_api_swarm_test.go [hunk 1]
```diff
@@ -409,8 +408,8 @@ func (s *DockerSwarmSuite) TestAPISwarmLeaveRemovesContainer(c *check.C) {
 
 	waitAndAssert(c, defaultReconciliationTimeout, d.CheckActiveContainerCount, checker.Equals, instances+1)
 
-	c.Assert(d.SwarmLeave(false), checker.NotNil)
```
Comment (thaJeztah) [inline integration-cli/docker_api_swarm_test.go]: Ah, hmm.... got it.. that won't work

**Comment by olljanat** [comment]
Derek add label: rebuild/*

**Comment by olljanat** [comment]
@thaJeztah looks to be some issue on z ?

**Comment by thaJeztah** [comment]
Hm, yeah, let me try and restart, but could be an issue with those machines 🤔

**Comment by olljanat** [comment]
Derek add label: rebuild/powerpc

**Review by vdemeester — APPROVED** [review]
LGTM :heart_eyes_cat:

**Comment by thaJeztah** [comment]
@yongtang @AkihiroSuda PTAL 🙏

**Review by yongtang — APPROVED** [review]
LGTM

**Pull request merged**